js/Grid/Grouping.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Grouping with Grid widget for Syncfusion Essential JS
-description: How to enable grouping and its functionalities
+title: Grouping in JavaScript Grid Control | Syncfusion
+description: Learn about grouping support in Syncfusion Essential JS Grip control, its elements and more.
```

Comment:
JavaScript Grid Control - spelling mistake in control name.

Reply:
renamed